[
 {
  "author": "arfrever", 
  "date": "2008-09-20T20:00:53.779293Z", 
  "msg": [
   "Follow-up to r33215:", 
   "* subversion/libsvn_fs/fs-loader.c", 
   "(fs_new): Mark with 'static' identifier.", 
   "Found by: danielsh"
  ], 
  "revision_id": "873290"
 }, 
 {
  "author": "arfrever", 
  "date": "2008-09-20T19:54:23.763892Z", 
  "msg": [
   "Use fs_new() instead of deprecated svn_fs_new().", 
   "* subversion/libsvn_fs/fs-loader.c", 
   "(fs_new): Renamed from svn_fs_new().", 
   "(svn_fs_new): Wrap fs_new().", 
   "(svn_fs_create, svn_fs_open, svn_fs_upgrade, svn_fs_recover): Use fs_new()", 
   "instead of deprecated svn_fs_new()."
  ], 
  "revision_id": "873289"
 }, 
 {
  "author": "arfrever", 
  "date": "2008-09-20T19:17:34.336120Z", 
  "msg": [
   "Allow to compile against D-Bus 0.*.", 
   "* configure.ac", 
   "(GNOME Keyring): Check D-Bus version and append -DDBUS_API_SUBJECT_TO_CHANGE", 
   "to CPPFLAGS when D-Bus 0.* is used.", 
   "Patch by: Alexander Thomas <alexander@collab.net>", 
   "(Tweaked by me.)"
  ], 
  "revision_id": "873288"
 }, 
 {
  "author": "danielsh", 
  "date": "2008-09-20T09:19:18.153976Z", 
  "msg": [
   "Following up to r33200, update a reference to a renamed table.", 
   "* subversion/libsvn_wc/wc-metadata.sql3", 
   "(I_PROPS_CHANGES):  Use the correct table name."
  ], 
  "revision_id": "873287"
 }, 
 {
  "author": "danielsh", 
  "date": "2008-09-20T08:18:10.379456Z", 
  "msg": [
   "* subversion/tests/cmdline/copy_tests.py", 
   "(test_list):", 
   "Mark find_copyfrom_information_upstairs() XFail on Windows.  (With the", 
   "r33201 fix, it doesn't hang -- it only errors -- hence \"XFail\" and not", 
   "\"Skip\".)"
  ], 
  "revision_id": "873286"
 }
]